src/gamepads.ts

```diff
@@ -0,0 +1,71 @@
+export const GAMEPAD_MAPS = {
+	// Generic gamepads
+	"Generic   USB  Joystick   (Vendor: 0079 Product: 0006)": {
```

Comment:
lol the `gamepad.id` have all these extra spaces?

Reply:
yep yep